pkg/webhook/validating/handler.go

```diff
@@ -123,20 +123,21 @@ func (h *WebhookHandler) HandleReviewRequest(path string, body []byte) (*v1.Admi
 	return response, nil
 }
 
-func DetectConfigurationAndWebhook(path string) (configurationId string, webhookId string) {
+// DetectConfigurationAndWebhook extracts configurationID and a webhookID from the url path.
+func DetectConfigurationAndWebhook(path string) (configurationID string, webhookID string) {
 	parts := strings.Split(path, "/")
 	webhookParts := []string{}
 	for _, p := range parts {
 		if p == "" {
 			continue
 		}
-		if configurationId == "" {
-			configurationId = p
+		if configurationID == "" {
+			configurationID = p
 			continue
 		}
 		webhookParts = append(webhookParts, p)
 	}
-	webhookId = strings.Join(webhookParts, "/")
+	webhookID = strings.Join(webhookParts, "/")
```

Comment:
Using `path` is more accurate.

Reply:
This webhookID is used as a key later in hook_manager, so the separator string should be constant.

Reply:
That is exactly what the path package provides.

Reply:
> // Package path implements utility routines for manipulating slash-separated
// paths.

https://github.com/golang/go/blob/master/src/path/path.go#L5-L6

I thought that `path` uses a system-dependent separator. Anyway, path.Join does a huge redundant [Clean](https://github.com/golang/go/blob/master/src/path/path.go#L70). I would rather use SplitN and drop append+Join.

Reply:
Ok, that sounds reasonable.